### ISSUE TITLE
npm version 1.0.13 is broken

### ISSUE THREAD

**Issue by sreimers** [n1c/vue3-avataaars#3]
Thanks for providing this library, looks like the last version is broken (v1.0.12 works fine). 

```
[plugin vite:dep-scan] Failed to resolve entry for package "vue3-avataaars". The package may have incorrect main/module/exports specified in its package.json.                                                                         
```

**Comment by n1c** [comment]
Oops! Sorry about that, fixed up in 1.0.14.

**Comment by sreimers** [comment]
Thanks!
